### DISCUSSION TITLE
Multiple modules in my quill

### DISCUSSION THREAD

**Discussion by toby044** [vueup/vue-quill#427] [Q&A]
I am trying to create a quill with multiple modules, but i am not understanding correctly how to do this.

In my app.js i set QuillEditor as a global component, and i can use it with blotformatter, where i do like this: 
```
data() {
	return {
		modules: {
			name: 'blotFormatter',
			module: BlotFormatter,
			options: {}
		}
	}
},

			<QuillEditor ref="quill" @ready="quillReady" theme="snow" :modules="modules" toolbar="full" v-model:content="quillData" contentType="html" />
```

I would like to also use quill-mention, but it gives me a registration error, when i try to use multiple like this: 

````
modules: {
	blotFormatter: {
		name: 'blotFormatter',
			module: BlotFormatter,
				options: {}
			},
	mention: {
		name: 'mention',
		module: Mention, 
		options: {}
	}
}

```

How can i properly use 2 modules on the same vue-quill

**Comment by azamat-747** [comment]
Same problem

**Reply by toby044** [reply]
I finished it a couple of weeks ago, but solved it by passing all the modules in as objects in an array instead, and imported them as object, which seemed to get the right information i wanted. I got errors that it created 2 instances of the quill, but it didnt show in the inspector as elements or impacted performance, so i disabled errors in my app.js
